src/ui/public/agg_types/metrics/bucket_avg.js

```diff
@@ -13,6 +14,17 @@ export function AggTypesMetricsBucketAvgProvider(Private) {
     subtype: siblingPipelineHelper.subtype,
     params: [
       ...siblingPipelineHelper.params()
-    ]
+    ],
+    getValue: function (agg, bucket) {
+      const customMetric = agg.params.customMetric;
+      const scalingMetric = customMetric.type && (customMetric.type.name === 'count' || customMetric.type.name === 'sum');
```

Comment:
I copied that line from https://github.com/elastic/kibana/blob/master/src/ui/public/agg_types/buckets/date_histogram.js#L116.  I'm not sure why Kibana only scales for counts and sums so I just left it as is.